app/views/users/_form.html.slim

```diff
@@ -0,0 +1,28 @@
+= simple_nested_form_for(@user) do |f|
+  = f.error_notification
+
+  .form-inputs
+    = f.input :first_name
+    = f.input :last_name
+    = f.input :email
+    = f.simple_fields_for :skills do |fs|
+      = render 'skill_form', fs: fs
+    = f.link_to_add 'Add a skill', :skills
+
+  .form-actions
+    = f.button :submit
+
+coffee:
+  $('input[name="user[first_name]"]').on 'change', ->
```

Comment:
You can use `= javascript_include_tag "application"` where `"application"` is the name of the manifest file you want to include e.g. `app/assets/applicaiton.js`